dev-ml/camlp4/camlp4-4.05_p1.ebuild

```diff
@@ -1,10 +1,10 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
And in this ebuild there are no functional changes at all.

Reply:
repoman auto fixed the lines